pkg/apis/infinispan/v1/types_util.go

```diff
@@ -428,11 +428,11 @@ func (ispn *Infinispan) ApplyOperatorLabels() error {
 	var errStr string
 	err := applyLabels(ispn, OperatorTargetLabelsEnvVarName, OperatorTargetLabels)
 	if err != nil {
-		errStr = fmt.Sprintf("Error unmarshalling "+OperatorTargetLabelsEnvVarName+" envvar: %v\n", err)
+		errStr = fmt.Sprintf("Error unmarshalling %s environment variable: %v\n", OperatorTargetLabelsEnvVarName, err)
```

Comment:
Is the "\n" character necessary?

Reply:
I don't know as this code was originally implemented by @rigazilla . I just add a fix for using `fmt.Sprintf` for all vars

Reply:
yeah it's not needed

Reply:
Thanks, @rigazilla will remove trailing `\n`